modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1252 @@
+// https://notion.so/tinycloud/Four-words-to-live-by-96e307110aaa4137b94f9ba12874d480
+
+=== Templates will be parsed before preview and insertion to make preview consistent with inserted template content and prevent XSS
+
+// https://ephocks.atlassian.net/browse/TINY-9244
+In previous versions of {productname}, templates were being parsed before `preview` and `insertion` to make preview consistent with content being inserted into the template to prevent Cross-Site Scripting (XSS) issues.
+As a consequence, the Templates plugin would not automatically format or sanitise the HTML passed to it as templates. This means that when previewing templates, the content may look different compared to when it is actually inserted into the editor. This can be a security issue, as unsanitised HTML can allow for Cross-Site Scripting (XSS), which can be harmful.
+
+To prevent this from happening, {productname} made improvements to the Template plugin that now sanitises content when retrieving preview content, and again before running `editor.dom.create` on it during the template insertion action. By doing this, all preview content will now match the template when it is inserted into the editor, which removed the possibility of any XSS issues arrising.
+
+In {productname} 6.4, Templates are now parsed before preview and insertion to make preview consistent with inserted template content and prevent XSS from occuring.
+
+// https://notion.so/tinycloud/Four-words-to-live-by-96e307110aaa4137b94f9ba12874d480
+
+=== Pressing backspace in an empty line now preserves formatting in a previous empty line
+
+// https://ephocks.atlassian.net/browse/TINY-9454
+In previous versions of {productname}, pressing `backspace` in an empty line would not preserve previously applied formatting to the previous line.
+
+As a consequence, when backspacing from an empty block from another empty block within {productname}, the editor would lose the preserved formatting for the previous block element.
+Its worth noting that `_this action is only excpected for when content is inserted into a `br[data-mce-bogus]` empty block. This is because the br[data-mce-bogus] tag is programmed to reset the formatting to the default setting for a empty block such as the `br`.
+
+[TIP]
+To avoid this, it is *important* to use a `backspace` instead of an `enter key` when moving from one empty block to another. This will ensure that the formatting of the previous block is preserved and the `br[data-mce-bogus]` tag is not inserted.
+
+To fix this issue, {productname} added improvements that now allow merge blocks to filter all block element decentants, such as the `br[data-mce-bogus]` block element, to ensure preservation of previously applied formatting is not removed from other block elements.
+
+In {productname} 6.4, pressing backspace in an empty line now preserves formatting in a previous empty line.
+
+// https://notion.so/tinycloud/Four-words-to-live-by-96e307110aaa4137b94f9ba12874d480
```

Comment:
```suggestion
In previous versions of {productname}, when hitting backspace, the deletion handler was found to be merging the current selected new line and the line above into one, by removing both, which resulted in them both being inserted then into a new line. 

Improvements were made to preserve the formatting when the backspace is "clicked",  from one empty block to another.

In {productname} 6.4, pressing backspace in an empty line now preserves the formatting in the previous empty line block.
```

Reply:
@mkzhx @TheSpyder Please review the above suggested change, and advise if you are happy for me to replace all suggestions above.

Reply:
```suggestion
In previous versions of {productname}, when pressing backspace in an empty line with another empty line containing inline formats above it, the deletion handler was found to be first removing both lines and then inserting a newline. As a consequence, the existing inline formats of the previous empty line was not preserved.

In {productname} 6.4, pressing backspace in an empty line now preserves the inline formats of the previous empty line.
```